### PR TITLE
Documentation Standard Updates

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
This pull request was created automatically 🎉:

- [x] Ensure `.github/` files have been properly updated (templates are updated and auto releaser is included)
- [x] Confirm all appropriate end models are included in the `.quickstart/quickstart.yml` file as `public_models` (only for transform repos)
- Verify the README has made all necessary updates:
    - [x] Removal of emojis
    - [x] Replacing exclamation points with periods in sentences
    - [x] Adjusting all headers to ensure only one H1 heading
    - [x] Confirming a section header is included for Quickstart UI rendering (only for transform repos)
    - [x] Renaming `**model` to `**table**` (only for transform repos)
    - [x] Removal of Fivetran community and office hour resources (if necessary)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -61,6 +67,19 @@ body:
         - other (mention it in "Additional Context")
     validations:
       required: true
+  - type: dropdown
+    id: orchestration_type
+    attributes:
+      label: How are you running this dbt package?
+      multiple: true
+      options:
+        - Quickstart
```
Comment (fivetran-joemarkiewicz) [inline .github/ISSUE_TEMPLATE/bug-report.yml]:
```suggestion
        - Fivetran Quickstart Data Model
```

File: README.md [hunk 2]
```diff
@@ -11,42 +11,45 @@
 </p>
 
 # Amazon Ads dbt Package ([Docs](https://fivetran.github.io/dbt_amazon_ads/))
-# 📣 What does this dbt package do?
+## What does this dbt package do?
 - Produces modeled tables that leverage Amazon Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/amazon-ads) in the format described by [this ERD](https://fivetran.com/docs/applications/amazon-ads#schemainformation) and builds off the output of our [Amazon Ads source package](https://github.com/fivetran/dbt_amazon_ads_source).
 - Provides insight into your ad performance across the following grains:
-  - Account, portfolio, campaign, ad group, ad, keyword, and search term
+- Account, portfolio, campaign, ad group, ad, keyword, and search term
```
Comment (fivetran-joemarkiewicz) [inline README.md]:
```suggestion
  - Account, portfolio, campaign, ad group, ad, keyword, and search term
```

File: README.md [hunk 3]
```diff
@@ -11,42 +11,45 @@
...
+## What does this dbt package do?
 - Produces modeled tables that leverage Amazon Ads data from [Fivetran's connector](https://fivetran.com/docs/applications/amazon-ads) in the format described by [this ERD](https://fivetran.com/docs/applications/amazon-ads#schemainformation) and builds off the output of our [Amazon Ads source package](https://github.com/fivetran/dbt_amazon_ads_source).
 - Provides insight into your ad performance across the following grains:
   - Account, portfolio, campaign, ad group, ad, keyword, and search term
 - Materializes output models designed to work simultaneously with our [multi-platform Ad Reporting package](https://github.com/fivetran/dbt_ad_reporting).
 - Generates a comprehensive data dictionary of your source and modeled Amazon Ads data through the [dbt docs site](https://fivetran.github.io/dbt_amazon_ads/).
 
-The following table lists all models that are materialized within this package by default. 
+<!--section="amazon_ads_transformation_model"-->
+The following table lists all models that are materialized within this package by default.
```
Comment (fivetran-joemarkiewicz) [inline README.md]:
```suggestion
The following table lists all tables that are materialized within this package by default.
```

**Review by fivetran-avinash — APPROVED** [review]
small change + SECRET added but good to go

**Pull request merged**